README.rst

```diff
@@ -84,13 +84,18 @@ is the directory with custom plugins of oh-my-zsh `(read more) <https://github.c
 
     git clone "https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv.git" "$ZSH_CUSTOM/plugins/autoswitch_virtualenv"
 
-
 Then add this line to your ``.zshrc``
 
 ::
 
     plugins=(autoswitch_virtualenv $plugins)
 
+Manual Installation
+
+::
+
+   source .../zsh-autoswitch-virtualenv/autoswitch_virtualenv.plugin.zsh
```

Comment:
this path seems very specific to the layout on your machine. Maybe something more obvious like 

`source $HOME/zsh-autoswitch-virtualenv/autoswitch_virtualenv.plugin.zsh`

Reply:
The triple dot was meant to represent whereever the user had installed but I have changed it to $HOME as suggested